packages/core/src/__tests__/unit/application.unit.ts

```diff
@@ -441,7 +441,9 @@ describe('Application', () => {
       });
     });
 
-    function logInterceptor(ctx: InvocationContext, next: Next) {}
+    function logInterceptor(ctx: InvocationContext, next: Next) {
+      return undefined;
```

Comment:
I find this redundant and a bit silly, because in JavaScript, functions return `undefined` when there is no `return` statement. I guess the extra verbosity is worth the stronger type safety your change provides.